src/MissionManager/QGCMapPolylineVisuals.qml

```diff
@@ -152,6 +158,14 @@ Item {
         }
     }
 
+    Component {
+        id: editPositionDialog
+        EditPositionDialog {
+            coordinate:  mapPolyline.vertexCoordinate(menu._removeVertexIndex)
```

Comment:
mapPolyline.path[menu._removeVertexIndex] should give you the same things without needing to change the signature for vertexCoordinate.

Reply:
yes ， I miss it